Design_Hashmap.java

```diff
@@ -0,0 +1,62 @@
+// Time Complexity : O(1)
```

Comment:
The time complexity in this implementation is always O(n)

Reply:
Thanks, I got it using" in" function always have O(N) time complexity. 
Same issue arises in Design2 Hashset question (I realized since running time was too big) I did not able to got any O(1) time complexity solution.